pyzeebe/credentials/oauth.py

```diff
@@ -0,0 +1,182 @@
+import json
+import logging
+import time
+import timeit
+from collections.abc import Callable
```

Comment:
Will change this to `from collections.abc import Callable`.
https://docs.python.org/3/library/typing.html#callable